lib/src/lib/page-header/components/page-header/page-header.component.ts

```diff
@@ -30,9 +30,9 @@ export class Lab900PageHeaderComponent implements OnInit {
   @Input()
   public breadCrumbs: BreadCrumb[];
 
-  public ngOnInit(): void {
-    this.leftActions = this.actions.filter((action) => action.align === 'left');
-    this.rightActions = this.actions.filter((action) => action.align === 'right' || action.align == null);
+  public ngOnChanges(): void {
+    this.leftActions = this.actions?.filter((action) => action.align === 'left');
```

Comment:
checken of dat this.actions effectief de changes is. 

  ```
public ngOnChanges(changes: SimpleChanges): void {
    if (changes.actions) {...}
  }  

```

Reply:
I was a bit too fast, fixed it